src/libnnstreamer-edge/nnstreamer-edge-internal.c

```diff
@@ -1144,11 +1221,53 @@ nns_edge_connect (nns_edge_h edge_h, nns_edge_protocol_e protocol,
   }
 
   eh->protocol = protocol;
+  if (0 != g_ascii_strcasecmp (eh->topic, "TCP_DIRECT")) {
```

Comment:
Suggestion)
"TCP_DIRECT" is used in both implementation code and test code. If then, what about making a header file for common usage?

Reply:
Good job! :1st_place_medal: